.github/workflows/main.yml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - master
+      - main
```

Comment:
Out of curiosity, are we planning on renaming master to main at any point?

Reply:
I don't have any strong opinion on it.  If someone wants to take the lead on seeing if that's a thing the community wants, I don't have any issue with it.  If you like, feel free to make an issue about it, or post to the contributors team, and we can figure out what people think. :+1:  (I don't imagine anyone would have a problem with it, but good to find out first.)